satellite/analytics/service.go

```diff
@@ -101,6 +118,54 @@ func (service *Service) enqueueMessage(message segment.Message) {
 	}
 }
 
+func (service *Service) createHubSpotUser(data PayloadUser) {
+
+	payloadBytes, err := json.Marshal(data)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	body := bytes.NewReader(payloadBytes)
+
+	req, err := http.NewRequest("POST", "https://api.hubapi.com/crm/v3/objects/contacts?hapikey=1da42564-0598-4456-a1dd-23bccbbb3c39", body)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	defer resp.Body.Close()
+
+}
+
+func (service *Service) sendHubSpotEvent(data_event interface{}) {
+	payloadBytes_event, err := json.Marshal(data_event)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	body_event := bytes.NewReader(payloadBytes_event)
+
+	req_event, err := http.NewRequest("POST", "https://api.hubapi.com/events/v3/send?hapikey=1da42564-0598-4456-a1dd-23bccbbb3c39", body_event)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	req_event.Header.Set("Content-Type", "application/json")
+
+	resp_event, err := http.DefaultClient.Do(req_event)
```

Comment:
the convention in the code base is to use camel case for variable names, e.g. `respEvent` instead of `resp_event`
also I think a lot of these variables would be fine as just `req`, `body`, `resp`, etc..